scripts/release/util/test_package.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+# TODO: use `trap` instead of cleanup function?
+
+GREEN_FG=$(tput setaf 2 2>/dev/null)
+RED_FG=$(tput setaf 1 2>/dev/null)
+TEAL_FG=$(tput setaf 6 2>/dev/null)
+END_FG_COLOR=$(tput sgr0 2>/dev/null)
+
+# TODO: The following error happens on centos:8
+#
+# Error:
+#  Problem: conflicting requests
+#    - nothing provides yum-cron needed by algorand-2.0.4-1.x86_64
+# (try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
+# smoke_test.sh: line 47: algod: command not found
+
+OS_LIST=(
+    centos:7
+#    centos:8
+    fedora:28
+    ubuntu:16.04
+    ubuntu:18.04
+)
+
+CHANNEL=stable
+FAILED=()
+
+while [ "$1" != "" ]; do
+    case "$1" in
+        -b)
+            shift
+            BRANCH="$1"
+            ;;
+        -c)
+            shift
+            CHANNEL="$1"
+            ;;
+        -h)
+            shift
+            HASH="$1"
+            ;;
+        -r)
+            shift
+            RELEASE="$1"
+            ;;
+        *)
+            echo "$RED_FG[$0]$END_FG_COLOR Unknown option $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
```

Comment:
same as before; make sure that these arguments are set before proceeding further.

Reply:
Will do.